src/elements/emby-slider/emby-slider.js

```diff
@@ -190,10 +190,7 @@ function setMarker(range, valueMarker, marker, valueProgress) {
             return;
         }
 
-        let markerPos = (bubbleTrackRect.width * valueMarker / 100) - markerRect.width / 2;
-        markerPos = Math.min(Math.max(markerPos, - markerRect.width / 2), bubbleTrackRect.width - markerRect.width / 2);
-
-        marker.style.left = markerPos + 'px';
+        marker.style.left = 'calc(' + valueMarker + '% - ' + markerRect.width / 2 + 'px)';
```

Comment:
nit: this would be easier to read, and probably a bit safer, with template literals
```suggestion
        marker.style.left = `calc(${valueMarker}% - ${markerRect.width / 2}px)`;
```

Template literals ensure the values are strings, and help avoid unintentional math. This is especially true in cases like this where the string itself contains arithmetic characters, so using `+` for concatenation is a little less clear.